.drone.yml

```diff
@@ -0,0 +1,81 @@
+workspace:
+  base: /var/www/owncloud
+  path: apps/user_ldap
+
+pipeline:
+  ldap:
+    image: dinkel/openldap
```

Comment:
What made you use this ldap container - compared to `osixia/openldap` which we use in other infrastructure projects?

Reply:
this script https://github.com/owncloud/administration/blob/master/ldap-testing/start.sh

Reply:
Ah didn't find these before :'(